README.md

```diff
@@ -125,6 +125,34 @@ Our markdown comes with metadata - also called "frontmatter". Here are the speci
 * **beta: true** - When this value is **true** a beta label is added to the page title.
 * **experimental: true** - When this value is **true** an experimental label is added to the page title.
 
+## DO NOT BUILD sections
+
+If you want to commit work, but
+- you do not want that work to be live on the public documentation site,
+- and also do not want to have it be in a **draft** status:
+
+You can set that section to not be built.
+
+Reasons you may want this would be for ongoing work for upcoming versions where we are still working out complications in the documentation, the feature or the documentation are incomplete but you want to commit work so you can continue to collaborate on this or other projects, features that are built for a certain slice of customers but not for all and not yet released, early beta work where things may still change, more time to review, etc.
+
```

Comment:
Probably don't need to explain all this since you explained it above in L131-132
```suggestion
```

Reply:
good point.